detox/test/src/Screens/DatePickerScreen.js

```diff
@@ -1,6 +1,10 @@
 import moment from 'moment';
 import React, { Component } from 'react';
-import { Text, View, StyleSheet, DatePickerIOS } from 'react-native';
+import { Text, View, StyleSheet, Platform, Button } from 'react-native';
+import { default as DatePicker, DateTimePickerAndroid } from "@react-native-community/datetimepicker";
```

Comment:
@mauricedoepke I'm wondering 🧐 does their default `<DatePicker>` work on Android? What is the difference between `DatePicker` and `DateTimePickerAndroid`? 

Could you consider adding tests for both components if they are both implemented on Android?

Reply:
So how I understand it, it has to do with the IOS native DatePicker being a regular "component" that you can inline anywhere you want in your UI like a regular react component.

The native Android DatePicker however apparently is implemented as a dialog which is called by an imperative API by default.

DatePicker under the hood calls DateTimePickerAndroid on android, so I think it suffices if we just test one method, because we are testing detox here and not react-native-datetimepicker/datetimepicker

But I will try to adapt it so that both ios and android use DatePicker.